draft-ietf-tls-tls13.md

```diff
@@ -631,7 +629,7 @@ using those algorithms over the contents of the element. The contents
 themselves do not appear on the wire but are simply calculated. The length of
 the signature is specified by the signing algorithm and key.
 
-In previous versions of TLS, the ServerKeyExchange format meant that attackers
+In previous versions of TLS, the ServerKeyShare format meant that attackers
```

Comment:
This actually needs to still read ServerKeyExchange, since it was called that there.